playbooks/force-to-enable-services.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+---
+- name: Install nodepool master node
+  become: yes
+  hosts: nodepool01
+  tasks:
+    - name: Force to enable nodepool services
+      shell: |
+        set -xe
+        systemctl enable nodepool-launcher.service || true
+        systemctl enable nodepool-launcher.service || true
+        systemctl enable nodepool-builder.service || true
+        systemctl enable zookeeper.service || true
```

Comment:
zookeeper service in master node? I see you enable below in zk03.

Reply:
yeah, they construct a cluster

Reply:
ok, thanks.